app/views/contexts/email_context.py

```diff
@@ -0,0 +1,20 @@
+from flask import url_for
+
+
+def build_confirmation_email_context(email_confirmation_form):
+    context = {
+        "hide_signout_button": False,
+        "sign_out_url": url_for("session.get_sign_out"),
+    }
+    context.update(build_email_form_context(email_confirmation_form))
+    return context
+
+
+def build_email_form_context(email_confirmation_form):
+    return {
+        "form": {
+            "mapped_errors": email_confirmation_form.map_errors(),
+            "fields": email_confirmation_form,
```

Comment:
This is setting `fields` to the wtforms form. The `question` context sets this property as a map of answer ids to their associated wtform field.